enterprise/errors.py

```diff
@@ -31,3 +31,13 @@ class UnlinkUserFromEnterpriseError(Exception):
     """
     An error occurred while unlinking a user from an enterprise.
     """
+
+class ClientError(Exception):
```

Comment:
i'd consider renaming this and/or namespacing this inside the SSO client/work work so its clearly the "Enterprise SSO" ClientError and not any of the other ones.

you may also want this to extend a `requests` Exception? duno